src/components/questions/MultipleChoiceList.tsx

```diff
@@ -1,66 +1,158 @@
-    .values;
+  const multipleChoiceOptions: string[] = (
+    props.questionOption.extra_data as MultipleChoiceExtraData
+  ).values;
+  const multipleAnswersPossible: boolean = (
+    props.questionOption.extra_data as MultipleChoiceExtraData
+  ).multiple;
+  const [currentValues, setCurrentValues] = useState(props.values ?? []);
+  let fadeAnim = new Animated.Value(0);
+
+  const selectOption = (value: any, index: number) => {
+    fadeAnim.setValue(index);
+    if (currentValues.includes(value)) {
+      value = currentValues.filter((currentValue) => {
+        return currentValue !== value;
+      });
+      setCurrentValues(value);
+      // fadeOutAnimation();
+    } else if (multipleAnswersPossible || currentValues.length === 0) {
+      value = [...currentValues, value];
+      setCurrentValues(value);
+      // fadeInAnimation();
+    }
+    props.onClicked(value);
+  };
+
+  // const fadeInAnimation = () => {
+  //   Animated.timing(fadeAnim, {
+  //     toValue: 0,
+  //     duration: 0,
+  //     useNativeDriver: true,
+  //   }).start(() => {
+  //     Animated.timing(fadeAnim, {
+  //       toValue: 1,
+  //       duration: 500,
+  //       delay: 10,
+  //       useNativeDriver: true,
+  //     }).start();
+  //   });
+  // };
+  //
+  // const fadeOutAnimation = () => {
+  //   Animated.timing(fadeAnim, {
+  //     toValue: 1,
+  //     duration: 0,
+  //     useNativeDriver: true,
+  //   }).start(() => {
+  //     Animated.timing(fadeAnim, {
+  //       toValue: 0,
+  //       duration: 500,
+  //       delay: 10,
+  //       useNativeDriver: true,
+  //     }).start();
+  //   });
+  // };
 
   return (
-    <RadioButtonRN
-      initial={
-        props.values && props.values.length > 0
-          ? multipleChoiceQuestions.indexOf(props.values[0]) + 1
-          : -1
-      }
-      data={stringArrayToRadioButtonData(multipleChoiceQuestions)}
-      selectedBtn={(selected: RadioButtonData) => {
-        props.onClicked(selected.label);
-      }}
-      animationTypes={['zoomIn', 'rotate']}
-      textStyle={styles.text}
-      style={styles.radioButton}
-      boxStyle={styles.box}
-      activeColor={COLORS.green}
-      deactiveColor={COLORS.darkBlue}
-      icon={<Icon name="check-circle" size={25} color={COLORS.green} />}
-    />
+    <View style={styles.multipleChoice} accessibilityLabel={'Meerkeuze'} accessible={true}>
+      {multipleChoiceOptions.map((value, index) => {
+        const isSelected = currentValues.includes(value);
+        return (
+          <TouchableOpacity
+            key={index}
+            style={[
+              styles.optionBox,
+              {
+                backgroundColor: isSelected ? '#e1f5fe33' : '#fff',
+                borderColor: isSelected ? COLORS.green : COLORS.darkBlue,
+              },
+            ]}
+            activeOpacity={0.9}
+            onPress={() => selectOption(value, index)}
+            accessibilityLabel={value + (isSelected ? ', geselecteerd' : '')}
```

Comment:
ACC label is meestal: "{waarde} {hoeveelste in de lijst}/{totaal aantal} {geselecteerd}"

Reply:
ACC label is meestal: "{waarde} {hoeveelste in de lijst}/{totaal aantal} {geselecteerd}" Dit heb jij van de participanten te horen gekregen?

Reply:
Heb even de standaard opgezocht omdat ik wel te horen kreeg dat het niet duidelijk was dat het meerkeuze was.